app/scripts/controllers/preferences.test.js

```diff
@@ -41,6 +42,30 @@ describe('preferences controller', function () {
     sinon.restore();
   });
 
+  describe('useBlockie', function () {
```

Comment:
These tests used to be in `metamask-controller.test.js` even though they were testing the preferences controller. They were moved here.